core/App/utils/helpers.ts

```diff
@@ -29,6 +31,54 @@ export function hashToRGBA(i: number) {
   return '#' + '00000'.substring(0, 6 - colour.length) + colour
 }
 
+export function formatTime(time: Date, params?: { long?: boolean; format?: string }): string {
+  const getMonthKey = 'MMMM'
+  const momentTime = moment(time)
+  const monthKey = momentTime.format(getMonthKey)
+  const customMonthFormatRe = /M+/
+  const long = params?.long
```

Comment:
You can shorten this to `const { long, format } = params`

Reply:
I tried that but then I get this error for each property: `Property 'format' does not exist on type '{ long?: boolean | undefined; format?: string | undefined; } | undefined'`

Reply:
Ah ok, I guess because `params` is optional itself then it can be `undefined`.